sso-kit-client/core/src/AccessProps.d.ts

```diff
@@ -12,5 +12,5 @@
  * authentication is needed to access that route.
  */
 export type AccessProps = Readonly<{
-  requiresLogin?: boolean;
+  protectedRoute?: boolean;
```

Comment:
I'd rename this to `requireAuthentication` since then we check for `SingleSignOnContext.authenticated` to be true.
```suggestion
  requireAuthentication?: boolean;
```

Reply:
I agree. I renamed it.